src/main/java/org/broadinstitute/consent/http/resources/TDRResource.java

```diff
@@ -64,4 +87,68 @@ public Response getDatasetByIdentifier(@Auth AuthUser authUser, @PathParam("iden
             return createExceptionResponse(e);
         }
     }
+
+    @POST
+    @Consumes("application/json")
+    @Produces("application/json")
+    @Path("/dar/draft")
+    @PermitAll
+    public Response createDraftDataAccessRequest(
+        @Auth AuthUser authUser,
+        @Context UriInfo info,
+        @QueryParam("identifiers") String identifiers,
+        @QueryParam("projectTitle") String projectTitle) {
+      try {
+        // Ensure that the user is a registered DUOS and SAM user
+        User user = userService.findOrCreateUser(authUser);
+        if (Objects.isNull(identifiers) || identifiers.isBlank()) {
+          throw new BadRequestException("No dataset identifiers were provided");
+        } else {
+          List<String> identifierList = Arrays.stream(identifiers.split(","))
+                  .map(String::trim)
+                  .filter(identifier -> !identifier.isBlank())
+                  // this will filter duplicate identifier strings, ex. "DUOS-000594, DUOS-000594"
+                  .distinct()
+                  .toList();
+          List<Integer> aliasList = identifierList
+                  .stream()
+                  .map(Dataset::parseIdentifierToAlias)
+                  // this will filter duplicate aliases, ex. "DUOS-00593, DUOS-000593"
```

Comment:
minor nit, the aliases are the trimmed integer values
```suggestion
                  // this will filter duplicate aliases, ex. "593, 593"
```